setup.py

```diff
@@ -64,10 +64,11 @@
 
 base_dir = os.path.dirname(os.path.abspath(__file__))
 
-_date = '20241122'
```

Comment:
nits: Maybe adding a flag for using the stable version, and add the `.dev{_date}` conditionally. This might make future pin update using nightly easier.

Reply:
oops, github auto-merge got too happy. i'll upload a fixup PR